polaris-react/src/components/ActionList/utils/text-content.ts

```diff
@@ -0,0 +1,19 @@
+import type React from 'react';
+
+export function textContent(
+  elem?: React.ReactElement | JSX.Element | string,
+): string {
+  if (!elem) {
+    return '';
```

Comment:
I wonder if we should return null here instead of empty string. If a consumer passes in undefined or something that returns undefined they'll get a blank action list item, I think